auth/src/desktop/rpcs/auth_request.cc

```diff
@@ -77,5 +81,21 @@ AuthRequest::AuthRequest(::firebase::App& app, const char* schema,
   }
 }
 
+std::string AuthRequest::GetUrl() {
+  std::string emulator_url = ::firebase::auth::Auth::GetEmulatorUrl();
```

Comment:
I'm trying to make this api desktop only, but cannot figure out a way to do that. Jon wdyt?